manifests/install.pp

```diff
@@ -35,14 +35,14 @@
         content => "${cron_minute} ${cron_hour} * * * root cd /opt/salus/bin && ./salus 2>&1 | logger -t salus\n",
     }
 
-    # Set up the cron for uploading certificates to the WAF, if parameters are set
-    if $waf_cron_minute and $waf_cron_hour {
+    # Set up the cron for uploading certificates to the Dashboard, if parameters are set
+    if $cert_upload_cron_minute and $cert_upload_cron_hour {
         file { "/etc/cron.d/salus-waf":
```

Comment:
Do you also want to replace `-waf` from `salus-waf` in the same way and call it `salus-cert`?

Reply:
Good call, renamed to `salus-upload`